Add Seeds to Systems strategy document (#1044)
open-life-science/open-life-science.github.io: _posts/2025-06-30-seeds-to-system-strategy.md

These structured programs remain crucial for onboarding researchers into open science, and we have also been offering **customised training and mentorship support for various organisations**. 
We will continue to strengthen this service area, aiming to generate revenue, providing a sustainable funding stream for Open Seeds and Nebula, which currently rely on grants and sponsorships.

**What's Next?** Under "Seeds to Systems," we are developing **advanced training** for researchers and open science practitioners with projects ready to transition to the next stage.
This program will equip them with the skills and strategies needed to design compelling project proposals, secure funding, and acquire resources to sustain their work using open-source/science solutions.
A new **Seeds to Systems Curriculum** will be designed to directly leverage learnings and collaborations from Open Seeds.

This mission will be developed in alignment with the **Open Research** and **Open Incubator** mission areas, providing critical resources and a progression path for researchers.
More information on this will be shared later in the year.

### Open Research

> Since 2022, OLS has been actively involved in open science research. With a purpose to ground our work in evidence, we have published reports and papers under research projects, including:

Author: ```suggestion
Since 2022, OLS has been actively involved in [open science research]({% link open-research %}). With a purpose to ground our work in evidence, we have published reports and papers under research projects, including:
```

> * **Impact of open science training and mentoring**, a qualitative study funded by the Wellcome Trust Open Research Fund.

Author: ```suggestion
* **Impact of open science training and mentoring**, a qualitative study funded by the Wellcome Trust Open Research Fund (2022-2024).
```

**What's Next?** Research will be fundamental to "Seeds to Systems," ensuring our training and operations are rooted in evidence and best practices.
Currently, two key research projects have been scoped:

* **Leverage points for community and project sustainability** to understand the challenges and needs of researchers, and identify key learning modules, the "leverage points", for the Seeds to Systems curriculum.
* **Research Funding Ecosystem** to map and document resources our global research community can refer to as part of training for securing research funding and fostering project sustainability.

These projects will directly inform and shape our other missions, ensuring they address the real-world skills and resources needed to lead and manage sustainable projects.
This direct feedback loop demonstrates how Research will reinforce the other missions. 
We aim to build a more equitable and accessible research ecosystem through training, mentorship, and community building.

### Open Incubator

Turning an idea into a project is a significant step, but leading and managing it effectively for long-term impact is another challenge entirely.

> The OLS Open Incubator offers vital expertise and **Fiscal Hosting infrastructure**, providing community members with expert guidance, financial management, and legal support, especially during the critical early stages of open science projects or building an organisation.

Author: ```suggestion
The OLS's [Open Incubator]({% link open-incubator.md %}) offers vital expertise and **Fiscal Hosting infrastructure**, providing community members with expert guidance, financial management, and legal support, especially during the critical early stages of open science projects or building an organisation.
```
